akka-remote/src/main/scala/akka/remote/artery/ArterySubsystem.scala

```diff
@@ -57,6 +68,11 @@ private[remote] class ArterySubsystem(_system: ExtendedActorSystem, _provider: R
     else Future.successful(Done)
   }
 
+  def sendReply(to: Address, message: AnyRef) = {
```

Comment:
What I don't like here is that it abuses the highest layer to inject a message that concerns only the lower layer. I.e. violation of concerns.